src/exec.js

```diff
@@ -91,10 +73,10 @@ function execSync(cmd, opts, pipe) {
     child.execSync(execCommand, opts);
   } catch (e) {
     // Clean up immediately if we have an exception
-    try { common.unlinkSync(scriptFile); } catch (e2) {}
-    try { common.unlinkSync(stdoutFile); } catch (e2) {}
-    try { common.unlinkSync(stderrFile); } catch (e2) {}
     try { common.unlinkSync(codeFile); } catch (e2) {}
+    try { common.unlinkSync(paramsFile); } catch (e2) {}
+    try { common.unlinkSync(stderrFile); } catch (e2) {}
+    try { common.unlinkSync(stdoutFile); } catch (e2) {}
```

Comment:
We're not using `e2` anyway, might as well call it `e` (it won't overwrite the `e` from above).

Reply:
I think there's an eslint warning against that...

Reply:
Oh, right.